gikari/bismuth
# build: :package: package as an archive with install script

## Summary

Now along with the .kwinscript the tar.gz archive is also packaged. This archive cannot be used in the KCM, but used to install the script in a more user-friendly way - without using npm and other development tools. 

Also, better structure is used for the assets and build directory.

## Test Plan

1. Run `npm run script-install`
2. Build directory should be populated with 3 directories: `script`, `package`, `extracted-package`.
3. Script should be successfully installed and working as before.

## Related Issues

Ref #66 (Will be fixed when I provide the working instructions for the end users in the README).
